api/src/main/java/org/pmiops/workbench/workspaceadmin/WorkspaceAdminService.java

```diff
@@ -0,0 +1,15 @@
+package org.pmiops.workbench.workspaceadmin;
+
+import java.util.Optional;
+import org.pmiops.workbench.db.model.DbWorkspace;
+import org.pmiops.workbench.model.AdminWorkspaceCloudStorageCounts;
+import org.pmiops.workbench.model.AdminWorkspaceObjectsCounts;
+
+public interface WorkspaceAdminService {
```

Comment:
I don't love that we have DB objects and IDs in this interface, but looking at other Workspace code I think that is too big a hill to climb for today.  This fits our current state of the world.